### DISCUSSION TITLE
DataCap distribution expectations across multiple SPs

### DISCUSSION THREAD

**Discussion by dkkapur** [filecoin-project/notary-governance#319] [General]
When clients apply for DataCap, there is an understanding that the deals will be distributed among several storage providers. However, there are cases where early dealmaking is concentrated to a smaller amount of SPs. In the case that this is for an LDN based application / DataCap allocation, the process of subsequent DataCap tranches being allocated to the client address is not very flexible to the "phases" that a client may go through when onboarding data onto the network. 

An example: https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/36#issuecomment-962585014

What are ways in which granting of DataCap in tranches can be better nuanced to clients' needs at specific points in time / SP context? How do we help clients find reliable SPs beyond just https://plus.fil.org/miners and filrep.io?

**Comment by galen-mcandrew** [comment]
Similar situation: https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/79

**Comment by BBQFIL** [comment]
Is it too harsh for storage customers. They just want to store data cheaper, not to get a bad experience. This is not conducive to the development of filecoin.

**Comment by cryptowhizzard** [comment]
I have stated this before, but it might be wise to ask for a list of SP's used for the LDN before datacap is granted. It prevents problems like this, prevents abuse and makes it more easy to check or negotiate.

For the experience part I would say that the current rules don't make things difficult. Price pressure is a good thing as long as the quality matches expectations.

**Comment by gdragonHH** [comment]
硬件算力出海，专业技术团队 独立节点，欢迎大家加V了解：Hmcxxxxx
